sysutils/cfm/Portfile

```diff
@@ -3,18 +3,18 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        willeccles cfm 0.5.5 v
+github.setup        willeccles cfm 0.6.0 v
 categories          sysutils
 platforms           darwin
 license             MPL-2
-maintainers         {eccles.dev:will @willeccles}
+maintainers         {eccles.dev:will @WillEccles}
 description         Cactus File Manager
 long_description    Cactus File Manager is a simple and fast TUI file manager
 homepage            https://github.com/willeccles/cfm
```

Comment:
The `homepage` line can be removed because that is the default one for the github portgroup.

Reply:
Resolved in https://github.com/macports/macports-ports/pull/6508/commits/6d0fd9c6d89228cf9516a9cff7a59c579ff12d34